scripts/setup.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 sudo apt -y update
-sudo apt install -y pkg-config libsystemd-dev libdbus-glib-1-dev build-essential libelf-dev libseccomp-dev libclang-dev
\ No newline at end of file
+sudo apt install -y pkg-config libsystemd-dev libdbus-glib-1-dev build-essential libelf-dev libseccomp-dev libclang-dev libssl-dev
```

Comment:
Do we need libssl-dev? Or was the README out of sync?

Reply:
these are the requirements from youki, but it doesn't look like we needed it since CI was running with out it.  @Mossaka any ideas?

Reply:
If it's not needed in the CI, then I don't it's needed anymore.

Reply:
removed it